packages/contracts-bedrock/package.json

```
@@ -10,25 +10,27 @@
   ],
   "scripts": {
     "bindings": "cd ../../op-bindings && make",
-    "build": "npx nx build:contracts",
+    "build": "nx build:contracts",
     "prebuild:contracts": "./scripts/verify-foundry-install.sh",
     "build:contracts": "pnpm build:forge",
     "build:forge": "forge build",
     "build:differential": "go build -o ./scripts/differential-testing/differential-testing ./scripts/differential-testing",
     "build:fuzz": "(cd test-case-generator && go build ./cmd/fuzz.go)",
-    "autogen:invariant-docs": "ts-node scripts/invariant-doc-gen.ts",
+    "generate:addresses": "tsx scripts/generate-addresses.ts",
+    "generate:addresses:check": "pnpm generate:addresses && git diff --exit-code ./addresses.json && git diff --exit-code ./addresses.json",
```

Looks like the git diff was duplicated here.
We should just be able to do
```suggestion
    "generate:addresses:check": "pnpm generate:addresses && git diff --exit-code ./addresses.json",
```